karapace/kafka_rest_apis/consumer_manager.py

```diff
@@ -145,19 +146,6 @@ def _illegal_state_fail(message: str, content_type: str) -> None:
             message=message,
         )
 
-    @staticmethod
-    def _update_partition_assignments(consumer: KafkaConsumer):
-        # This is (should be?) equivalent to calling poll on the consumer.
-        # which would return 0 results, since the subscription we just created will mean
-        # a rejoin is needed, which skips the actual fetching. Nevertheless, an actual call to poll is to be avoided
-        # and a better solution to this is desired (extend the consumer??)
-
-        # pylint: disable=protected-access
-        consumer._coordinator.poll()
-        if not consumer._subscription.has_all_fetch_positions():
-            consumer._update_fetch_positions(consumer._subscription.missing_fetch_positions())
-        # pylint: enable=protected-access
```

Comment:
Is this removed because not needed with aiokafka or for some other reason?

Reply:
Yes.

1. This uses kafka-python's internal implementation, not available in aiokafka.
2. As far as I understood aiokafka automatically does needed partition assignments after `subscribe` or `assign` (the cases when this internal method was used with kafka-python)

Reply:
> This uses kafka-python's internal implementation, not available in aiokafka

Got it.